deck/forms.py

```diff
@@ -30,6 +30,11 @@ class Meta:
                 'class': 'inline-input',
                 'placeholder': 'Due Date'
             }),
+            'accept_proposals_at': CustomDateTimeWidget(attrs={
+                'id': 'id_accept_proposals',
+                'class': 'inline-input',
+                'placeholder': 'Date Accept Proposals'
```

Comment:
this needs to be translated.

Reply:
i writed this chunk of code, beacause when the template generates the html page, it puts inline-input as a class and Date Accept Proposals as placeholder to input tag

Reply:
What i mean was that you need to use the `ugettext` function. `_('Date Accept Proposals')`, it would be something like this.

Reply:
I'm sorry, I don't see the ugettext function